app/viewmodels/checkAnswers/euDetails/EuSendGoodsAddressSummary.scala

```diff
@@ -35,6 +35,7 @@ object EuSendGoodsAddressSummary {
         val changeLinkMode = currentMode match {
           case NormalMode    => CheckLoopMode
           case CheckMode     => CheckMode
+          case AmendMode     => AmendMode
```

Comment:
I'm wondering if we have to have a AmendLoopMode

Reply:
Fixed